src/traces/scatter3d/convert.js

```diff
@@ -124,9 +125,9 @@ function calculateErrorParams(errors) {
         if(e && e.copy_zstyle !== false) e = errors[2];
         if(!e) continue;
 
-        capSize[i] = e.width / 2;  // ballpark rescaling
+        capSize[i] = (e.width || 0) / 2;  // ballpark rescaling
         color[i] = str2RgbaArray(e.color);
-        lineWidth = e.thickness;
+        lineWidth = (e.thickness || errorBarsAttributes.thickness.dflt);
```

Comment:
Hmm. Strange, judging from

https://github.com/plotly/plotly.js/blob/e545415463730291201e0adadb876874680c4361/src/components/errorbars/defaults.js#L71

`fullData[i]._thickness` should have a set value by the time it gets here.

Reply:
Note that having values set in `fullData` is (very) important for the react-chart-editor (aka RCE). RCE uses `fullData` to fill in its UI components.